### PR TITLE
Initial working module

### PR DESCRIPTION
**Pull request by ChemiKyle**
To test:
1. Activate the module on any project which has had alert emails sent (e.g. a local clone of the FRCovid project).
1. Set **Field to pipe alert log** to an appropriate field (e.g. `result_comments`)
1. Check **Search all alerts on next run**
1. Wait at least 1 minute
1. Navigate to any record and event containing the field you specified (note the event must be used in at least one alert)
1. Observe that the comments field is now populated with "<alert_title> alert email sent: <timestamp>" lines

___

~~The **Hours back to search for alerts** option is a bit pointless in light of the **Search all alerts on next run** option. I'd like to remove it before this PR is merged unless there's a compelling argument to keep it. Also its name is bad.~~. 
Configuration removed in commit 5283f2a, now only the past 8 minutes of emails are checked (cron runs every minute, allowed to take a maximum of 6 minutes, 2 extra buffer minutes are allotted in the event of a one-off error)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-# copy_email_alert_log_to_field
-A REDCap module to copy email alert log entries into a REDCap Field
+# Copy Email Alert Log to Field
+A REDCap module to copy email alert log entries into a user-defined REDCap text field. All alerts for tied to an event will appear in the corresponding field on that event.
```
Comment (pbchase) [inline README.md]:
```suggestion
A REDCap module to copy email alert log entries into a user-defined REDCap text field. All alerts tied to an event will appear in the corresponding field on that event.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+
+## Installation
+- Clone this repo into to `<redcap-root>/modules/copy_email_alert_log_to_field_v0.0.0`.
+- Go to **Control Center > Manage External Modules** and enable Copy Email Alert Log to Field
+- For each project you want to use this module, go to the project home page, click on **Manage External Modules** link, and then enable Copy Email Alert Log to Field for that project.
+
+## Configuration
+
+- **Field to pipe alert log**: Select a field to pipe email alert logs in to. The chosen field **must exist** in every **event** you have an alert set for to capture alerts for that event.
+- **Search all alerts on next run**: Check this box to search the log of all alerts on the next run. Recommended to turn on only if enabling on a project which has already sent email alerts out. This turns itself off after running.
```
Comment (pbchase) [inline README.md]: The language here is a bit odd.  How about this instead? Did I get your intent?

```suggestion
- **Field to pipe alert log**: Select a field to write email alert logs into. The chosen field **must exist** in each REDCap event where you would like to capture alerts.
- **Search all alerts on next run**: Check this box to search the log of all alerts on the next run for alert history to copy into the field. This switch is recommended only if you are enabling this module on a project which has already sent email alerts out. As it should only be needed once, this switch turns itself off after running once. 
```

**Comment by pbchase** [comment]
Did you have a test project for this? I am happy to make one if not. I'm just being lazy if the can be :-)

**Comment by ChemiKyle** [comment]
@pbchase I was using a clone of one of the FRCovid projects with email alerts on and filling in my own data to trigger the send. Unfortunately my clone died some time ago when killed my 9.3.5 instance and its volumes.

**Comment by ChemiKyle** [comment]
\>:( hey I was doing that!

LOL!

**Pull request merged**